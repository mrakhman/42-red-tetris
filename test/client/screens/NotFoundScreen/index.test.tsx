import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import NotFoundScreen from '/client/screens/NotFoundScreen';

describe('Snapshots', () => {
  it('should render NotFoundScreen screen', () => {
    const wrapper = shallow(<NotFoundScreen />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
