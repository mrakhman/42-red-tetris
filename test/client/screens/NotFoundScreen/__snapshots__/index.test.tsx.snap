// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots should render NotFoundScreen screen 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
      "padding": 20,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 20,
        "fontWeight": "bold",
      }
    }
  >
    This screen doesn't exist.
  </Text>
  <ForwardRef
    onPress={[Function]}
    style={
      Object {
        "marginTop": 15,
        "paddingVertical": 15,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#2e78b7",
          "fontSize": 14,
        }
      }
    >
      Go to home screen!
    </Text>
  </ForwardRef>
</View>
`;
